### ISSUE TITLE
tackandlay icon in the toobar is too similar to VDR

### ISSUE THREAD

**Issue by dominig** [cowelld/tackandlay_pi#15]
The icon for tackandlay is too close of the tow used and established for the VDR plugin and this is confusing.
See screenshot.

![tackandlay-20160323-2055](https://cloud.githubusercontent.com/assets/1193120/13998939/0562c4c0-f13a-11e5-8054-cd275fad8386.jpg)

**Comment by rgleason** [comment]
I agree, why don't we make a nice simple one with Port and Starboard Laylines?
